doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -538,12 +594,12 @@ gls{component}
 \labelwidthstring 00.00.0000
 \begin_inset CommandInset label
 LatexCommand label
-name "/OF120/"
+name "/OF180/"
 
 \end_inset
 
  If requested by the user, Beagle shuts down the computer it's running on
- after it finished.
+ after it finished a measurement.
```

Comment:
“the computer” is very general and I tried to be precise. Is "If reqeusted by the user, Beagle shuts down the computer it’s running on after all measurements finished." okay?